saltproc/openmc_depcode.py

```diff
@@ -136,10 +143,10 @@ def run_depletion_step(self, cores, nodes):
             self.runtime_inputfile['geometry'],
             '--settings',
             self.runtime_inputfile['settings'],
-            '--tallies',
+            '--tallites',
             self.runtime_inputfile['tallies'],
-            '--depletion_settings',
-            self.runtime_inputfile['depletion_settings'])
```

Comment:
because now the depletion settings are an expected value, `depletion_settings.json`. See lines 252-254 in `openmc_depcode.py`:
```python
  with open(self.output_path / 'depletion_settings.json', 'w') as f:
             depletion_settings_json = json.dumps(self.depletion_settings, indent=4)
             f.write(depletion_settings_json)

```